account/src/main/java/bisq/account/AccountService.java

```diff
@@ -43,15 +43,15 @@ public class AccountService extends RateLimitedPersistenceClient<AccountStore> i
 
     private final AccountStore persistableStore = new AccountStore();
     private final Persistence<AccountStore> persistence;
-    private final transient ObservableSet<Account<?, ? extends PaymentMethod<?>>> accounts = new ObservableSet<>();
 
     public AccountService(PersistenceService persistenceService) {
         persistence = persistenceService.getOrCreatePersistence(this, DbSubDirectory.PRIVATE, persistableStore);
     }
 
     @Override
     public void onPersistedApplied(AccountStore persisted) {
-        accounts.setAll(persisted.getAccountByName().values());
+        persistableStore.getAccountByName().clear();
```

Comment:
The onPersistedApplied can be removed. The store gets the data set from the persistence framework.